src/stories/containers/Finances/components/DelegateExpenseTrend/DelegateExpenseTrendItem.tsx

```diff
@@ -14,7 +14,7 @@ import lightTheme from '@ses/styles/theme/themes';
 import { DateTime } from 'luxon';
 import Link from 'next/link';
 import React, { useMemo } from 'react';
-import ActorLastModified from '@/views/Actors/components/ActorLastModified/ActorLastModified';
+import LastModifiedActorCoreUnit from '@/components/LastModifiedActorCoreUnit/LastModifiedActorCoreUnit';
```

Comment:
if `LastModifiedActorCoreUnit` is a component spacific for the core unit index page, it should live in the `component` directory of the view folder instead of the global components folder

Reply:
this component its use for both, that its way its in the global folder (Core Unit and EA)